packages/eas-cli/src/commands/device/list.ts

```diff
@@ -0,0 +1,90 @@
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import ora from 'ora';
+
+import formatDevice from '../../devices/utils/formatDevice';
+import { AccountQuery } from '../../graphql/queries/AccountQuery';
+import { AppleDeviceQuery } from '../../graphql/queries/AppleDeviceQuery';
+import { AppleTeamQuery } from '../../graphql/queries/AppleTeamQuery';
+import log from '../../log';
+import { findProjectRootAsync, getProjectAccountNameAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+
+export default class BuildList extends Command {
+  static description = 'list all registered devices for your account';
+
+  static flags = {
+    'apple-team-id': flags.string(),
+  };
+
+  async run() {
+    let appleTeamIdentifier = this.parse(BuildList).flags['apple-team-id'];
+
+    const projectDir = (await findProjectRootAsync()) ?? process.cwd();
+    const accountName = await getProjectAccountNameAsync(projectDir);
+
+    let spinner: ora.Ora;
+
+    if (!appleTeamIdentifier) {
+      spinner = ora().start('Fetching the list of teams for the project…');
+
+      try {
+        const teams = await AppleTeamQuery.allForAccountAsync(accountName);
+
+        if (teams.length) {
+          spinner.succeed();
+
+          if (teams.length === 1) {
+            appleTeamIdentifier = teams[0].appleTeamIdentifier;
+          } else {
+            const result = await promptAsync({
+              type: 'select',
+              name: 'appleTeamIdentifier',
+              message: 'What Apple Team would you like to list devices for?',
+              choices: teams.map(team => ({
+                title: team.appleTeamName ?? team.id,
```

Comment:
I'm not sure what to show in the prompt if there is no name

Reply:
Maybe if there is team name display both values and if it's only id then only that one value.
- `teamId - Team Name`
- `teamID (Team Name)`

Reply:
My suggestion: `${team.appleTeamName} (ID: ${team.id})`